### PR TITLE
Changes for the charts on the dashboard

### PR DESCRIPTION
**Pull request by aarjavshah**
I have created two charts. I still need to discuss on the third chart with the team. We will discuss and put the actual data in it as it is needed. Two charts are working as expected.

### CODE REVIEW

File: src/actions/profileActions.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
+import { SET_STATUS_LIST } from "./prefillActions";
+
```
Comment (CGIBIFE) [inline src/actions/profileActions.js]: Please remove the space

File: src/reducers/profileReducer.js [hunk 2]
```diff
@@ -1,27 +1,69 @@
...
             }
+          }
 
         case CREATE_PROFILE:
             return {
                 ...state,
                 profile: action.data
             }
+
+        // case SET_STATS:
```
Comment (CGIBIFE) [inline src/reducers/profileReducer.js]: Please remove the commented code

File: src/components/Dashboard.js [hunk 3]
```diff
@@ -57,64 +59,47 @@ class Dashboard extends React.Component {
...
-                            rootProps={{"data-testid": "1"}}
+                            options={pieOptions}
+                            legend_toggle
+                            //rootProps={{"data-testid": "1"}}
                         />
                     </Col>
                     <Col>
                         <Chart
-                            width={"100%"}
+                            width={"110%"}
```
Comment (CGIBIFE) [inline src/components/Dashboard.js:91]: Why the change in value, is their data for this chart is available

Comment (aarjavshah) [reply src/components/Dashboard.js:91]: It is supposed to be picked up in upcoming week to display actual data to showcase the kind of data that is mocked up. This was changed as the bar chart looked too small compared to the pie chart

File: src/actions/profileActions.js (hunk 1 above)
Comment (aarjavshah) [reply src/actions/profileActions.js]: changed.

File: src/reducers/profileReducer.js (hunk 2 above)
Comment (aarjavshah) [reply src/reducers/profileReducer.js]: done.
